src/bosh_azure_cpi/spec/manual_tests/performance/run_performance.sh

```diff
@@ -0,0 +1,85 @@
+chruby 2.4.4
+source ./run_performance.pri.sh
+
+export SERVICE_HOST_BASE="core.windows.net"
+
+pushd tf
+terraform init
+
+export RESOURCE_GROUP=${PREFIX}-rg
+export VNET_NAME=${PREFIX}-vnet
+export SUBNET_NAME=${PREFIX}-bosh-subnet
+default_storage_account=$(echo $PREFIX | sed -e 's/[^0-9a-z]//g')
+default_storage_access_key=$(az storage account keys list -g ${PREFIX}-rg -n ${default_storage_account} | jq .[0].value)
+
+connection_string="DefaultEndpointsProtocol=https;AccountName=${default_storage_account};AccountKey=${default_storage_access_key};EndpointSuffix=${SERVICE_HOST_BASE}"
+
+az storage container create --name bosh --connection-string ${connection_string}
+az storage container create --name stemcell --public-access blob --connection-string ${connection_string}
+az storage table create --name stemcells --connection-string ${connection_string}
+
+terraform apply -var prefix="${PREFIX}" \
```

Comment:
Is the default storage account created by `terraform`? If so, the containers can't be created before running `terraform apply`.

Reply:
good catch.